README.md

```diff
@@ -21,7 +21,7 @@ dependencies {
     * [Making Cassandra more capable, faster, and more reliable](https://www.slideshare.net/scalar-inc/making-cassandra-more-capable-faster-and-more-reliable-at-apacheconhome-2020) at ApacheCon@Home 2020
     * [Scalar DB: A library that makes non-ACID databases ACID-compliant](https://www.slideshare.net/scalar-inc/scalar-db-a-library-that-makes-nonacid-databases-acidcompliant) at Database Lounge Tokyo #6 2020
     * [Transaction Management on Cassandra](https://www.slideshare.net/scalar-inc/transaction-management-on-cassandra) at Next Generation Cassandra Conference / ApacheCon NA 2019
-* [Javadoc](https://scalar-labs.github.io/scalardb/javadoc/)
+* [Javadoc](https://javadoc.io/doc/com.scalar-labs/scalardb/latest/index.html)
```

Comment:
I think we can/should also refer scalardb-rpc and scalardb-server?

I manually typed scalardb-rpc and scalardb-server, and they don't exist but it seems to trigger creation.
![Screen Shot 2021-08-06 at 10 34 04](https://user-images.githubusercontent.com/1052473/128442550-e4f9ddee-34dd-4f0c-b27c-616d575888fd.png)

Reply:
Sure. I will update it. Thank you!